tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1428,6 +1441,25 @@ process_params(void)
   }
 }
 
+/**
+   \brief Constructs needed vectorlength nodes
+ */
+static void construct_vectorlength_metadata(LL_MDRef* loop_md, int vectorlength_factor, int scalable)
```

Comment:
Keep pointer declaration formatting of the project, `LL_MDRef *loop_md` not `LL_MDRef* loop_md`.